power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/serialization/serializer.hpp

```diff
@@ -20,10 +20,8 @@
 #include <string_view>
 
 // custom packers
-namespace msgpack {
+namespace msgpack::adaptor {
 MSGPACK_API_VERSION_NAMESPACE(MSGPACK_DEFAULT_API_NS) {
-    namespace adaptor {
-
```

Comment:
this is a false positive. you need `MSPACK_API_VERSION_NAMESPACE` or everything will break

Reply:
Yep, I noticed that and was reverting it just now :)

Reply:
Although all the builds and tests pass

Reply:
what do you mean? do you mean all builds in CI? or the C++ ones? or the ones on your PC? there may be version-, compiler- and OS-dependencies here, hence the differences